src/PhpSpreadsheet/Reader/Xlsx/ColumnAndRowAttributes.php

```diff
@@ -91,6 +92,10 @@ public function load(?IReadFilter $readFilter = null, $readDataOnly = false): vo
             $rowsAttributes = $this->readRowAttributes($this->worksheetXml->sheetData->row, $readDataOnly);
         }
 
+        if ($readFilter !== null && get_class($readFilter) === DefaultReadFilter::class) {
+            $readFilter = null;
```

Comment:
If it makes a measurable difference, wouldn't it be better to simply have the defaultFilter as null in the BaseReader rather than defaulting it to the DefaultReadFilter?

Reply:
Since (as far as I could tell) every call-site of the read filter checks if it is null, that's probably a safe way to go.